src/CMakeLists.txt

```diff
@@ -1292,6 +1292,18 @@ else()
    target_link_libraries(findlib ${CMAKE_DL_LIBS})
 endif()
 
+# Common to all platforms: remove the imageCompiler program so it isn't needlessly
+# copied into program bundles
+if (TARGET image-compiler)
```

Comment:
I think we should just put this executable in a different location. This way it won't be installed on Windows/macOS and there will be no need to rebuild it constantly.